src/components/action-popover/action-popover.stories.mdx

```diff
@@ -536,18 +546,17 @@ a &quot;Manage&quot; option.
           <TableCell>
             <ActionPopover
               rightAlignMenu
-              renderButton={ ({ styleOverride, tabIndex, ...rest }) => (
-                <Button
+              renderButton={ ({ tabIndex, 'data-element': dataElement }) => (
```

Comment:
I'm not sure the `data-element` is transmitted well. I can't find the `data-element` in the DOM
![image](https://user-images.githubusercontent.com/19231884/80457361-73f31480-892f-11ea-91dc-3ca8d4eeaa32.png)

Reply:
good spot I have s typo in the stories

Reply:
fixed https://github.com/Sage/carbon/pull/2894/commits/feef60f75ae13aedfed3e8e4db8d4702a01c02c2